tests/admin.test.ts

```diff
@@ -182,3 +183,71 @@ describe('bonus points submission', () => {
     expect(getNoBonusUserResponse.user.points).toEqual(0);
   });
 });
+
+describe('event creation', () => {
+  test('successful event creation', async () => {
+    const conn = await DatabaseConnection.get();
+    const admin = UserFactory.fake({ accessType: UserAccessType.ADMIN });
```

Comment:
Actually lets follow the success test in attendance test 'only admins can see attendances for event' to also check for permissions.